app/services/idv/proofer.rb

```diff
@@ -11,7 +11,7 @@ module Proofer
 
     STAGES = %i[resolution state_id address].freeze
 
-    @vendors = {}
+    @vendors = nil
```

Comment:
Is this indicative of a missing spec?

Reply:
I thought it was at first, but the specs set this to `nil` before testing, so this just makes the code mirror what the specs are expecting it to be.